static/js/components/EducationTab.js

```diff
@@ -0,0 +1,249 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Button from 'react-mdl/lib/Button';
+import _ from 'lodash';
+
+import ProfileTab from "../util/ProfileTab";
+import { generateNewEducation } from "../util/util";
+import {
+  boundMonthField,
+  boundYearField,
+  saveAndContinue
+} from '../util/profile_edit';
+import {
+  toggleEducationLevel,
+  openEducationForm,
+  closeEducationForm
+} from '../actions';
+import { HIGH_SCHOOL, ASSOCIATE, BACHELORS, MASTERS, DOCTORATE } from '../constants';
+import { Grid, Cell, Switch, Dialog, DialogTitle, DialogContent, DialogActions, FABButton, Icon } from 'react-mdl';
+
+class EducationTab extends ProfileTab {
+  constructor(props) {
+    super(props);
+    this.boundMonthField = boundMonthField.bind(this);
+    this.boundYearField = boundYearField.bind(this);
+    this.educationLevelOptions = [
+      {value: HIGH_SCHOOL, label: "High school"},
+      {value: ASSOCIATE, label: 'Associate degree'},
+      {value: BACHELORS, label: "Bachelor's degree"},
+      {value: MASTERS, label: "Master's or professional degree"},
+      {value: DOCTORATE, label: "Doctorate"}
+    ];
```

Comment:
what do you think about moving this stuff into the constructor for `ProfileTab`? I think we may want the month and year fields for the work history tab, I've also been putting stuff like `educationLevelOptions` in the `ProfileTab` constructor so that these classes inheriting from `ProfileTab` end up looking more declarative (just saying what kinds of inputs should be drawn and so on)